cmd/juju/cloud/showcredential.go

```diff
@@ -49,6 +51,7 @@ func (c *showCredentialCommand) SetFlags(f *gnuflag.FlagSet) {
 		"yaml": cmd.FormatYaml,
 	})
 	f.BoolVar(&c.ShowSecrets, "show-secrets", false, "Display credential secret attributes")
+	f.BoolVar(&c.Local, "local", false, "Local operation only; controller not affected")
```

Comment:
Show credentials shouldn't "affect" the controller. I'd reword this to something like "Local operation only; controller credentials not listed"

Reply:
This is a standard description for all --local options.

Reply:
Right, but in most cases the command does something on/to the controller - here it's confusing. It's more important for it to make sense in the context of this command than to be consistent globally (when they pull in different directions).